manager.py

```diff
@@ -0,0 +1,126 @@
+import cassandra_repository
+import http_repository
+import dramatiq
+from dramatiq.brokers.rabbitmq import RabbitmqBroker
+
+
+rabbitmq_broker = RabbitmqBroker(url="amqp://guest:guest@rabbit:5672/")
+dramatiq.set_broker(rabbitmq_broker)
+
+db_rep = cassandra_repository
+network_rep = http_repository
+
+bad_chars = {',', '.', ':', ':', ')', '(', '{', '}'}
+
+common = {'a', 'an', 'and', 'are', 'as', 'at', 'be', 'but', 'by', 'for', 'if', 'in', 'into', 'is', 'it', 'no', 'not',
+          'of', 'on', 'or', 'such', 'that', 'the', 'their', 'then', 'there', 'these', 'they', 'this', 'to', 'was',
+          'will', 'with'}
+
+
+def init():
+    db_rep.init()
+
+
+def run_index_async():
+    index.send()
+
+
+@dramatiq.actor
+def index():
+    db_rep.go_to_keyspace()
+
+    res = network_rep.get_documents()
+    docs_set = set()
+    for doc in res['documents']:
+        if (doc['title'], doc['author']) not in docs_set:
+            docs_set.add((doc['title'], doc['author']))
+            index_document(doc)
+
+
+def index_document(doc):
+    counter = 0
+    words_dict = {}
+    words = doc['content'].split()
+    for idx, word in enumerate(words):
+        if '<' not in word:
+            fixed = fix_word(word)
```

Comment:
just replace word with fixed. also fixed is not good parameter name

Reply:
Done